public/reportissue.php

```diff
@@ -117,14 +117,14 @@ function displayCore() {
                     </td>
                 </tr>
                 <tr>
-                    <td><label for="checksum">Checksum:</label></td>
+                    <td><label for="checksum">ROM:</label></td>
                     <td>
                         <select name="note[checksum]" id="checksum" required>
                             <option value="Unknown">I don't know.</option>
                             <?php
                             foreach (getHashListByGameID($gameID) as $listKey => $hashArray) {
-                                foreach ($hashArray as $hashKey => $hash) {
-                                    echo "<option value='$hash'>$hash</option>";
+                                foreach ($hashArray as $hashKey) {
+                                    echo "<option value='$hashKey[\"hash\"]'>$hashKey[\"filename\"] ($hashKey[\"hash\"]) </option>";
```

Comment:
PHP is encountering a syntax error because of of the quoted array keys, so this code will not work.

It might be better to extract the key-value pairs into their own variables instead of trying to interpolate all of these quote characters into an HTML string, like so:

```
extract($hashKey, EXTR_PREFIX_ALL, "hk");
echo "<option value='$hk_hash'>$hk_filename ($hk_hash) </option>";
```

This will create variables for each of the array key-value pairs and prefix them with "hk" to avoid conflicts with any other variables that may have previously been defined.